rollingpin/commands.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABCMeta, abstractproperty
+
+
+class Command(object):
+    __metaclass__ = ABCMeta
```

Comment:
not gonna lie i'm not really sure if i need to do this. just want to follow best practices for a base type that should not be instantiated directly

Reply:
:man_shrugging: i usually wouldn't bother. if someone _did_ instantiate it directly they'd be promptly getting some `NotImplementedError`s anyway